### PR TITLE
Ui Changes.

### PR DESCRIPTION
**Pull request by cmkohnen**
- Redesign preferences
- Remove redundant subtitles
- Linguistic/Contextual improvements

### CODE REVIEW

File: data/ui/preferences.blp [hunk 1]
```diff
@@ -136,20 +122,19 @@ template $PreferencesWindow : Adw.PreferencesWindow {
               _("Lower center"), _("Upper center"), _("Center"),
             ]
           };
+        }
       }
-    }
 
       Adw.ExpanderRow figure_use_custom_style {
         title: _("Use Custom Style");
```
Comment (Sjoerd1993) [inline data/ui/preferences.blp:129]: It should be made clear somehow that this is to set the default style, upon launch of a new session. The style of the active window is set in Figure settings.

Perhaps "Use Custom Style by Default", or "Use Custom Style upon Launch", something like that.

**Review by Sjoerd1993 — CHANGES_REQUESTED** [review]
Overall good changes, I've just got the single comment about that with "Custom Style" in preferences it should be made clear that this sets the defaults, not the current active canvas.

File: data/ui/preferences.blp (hunk 1 above)
Comment (cmkohnen) [reply data/ui/preferences.blp]: Yes, absolutely

**Review by Sjoerd1993 — APPROVED** [review]
I like the new preference window, it's absolutely more clear and easy to find stuff.

**Pull request merged**